.gitignore

```diff
@@ -17,4 +17,5 @@ dist/
 .pytest_cache/
 
 # Other
-*.DS_Store
\ No newline at end of file
+*.DS_Store
```

Comment:
Please that .gitignore changes.

Reply:
bypass pycharm(popular Python IDE) configurations.

Reply:
done, please to check #739